packages/minicli/minicli.3.0.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "minicli"
+maintainer: "ligand@free.fr"
+authors: "Francois Berenger"
+homepage: "https://github.com/UnixJunkie/minicli"
+bug-reports: "https://github.com/UnixJunkie/minicli/issues"
+dev-repo: "https://github.com/UnixJunkie/minicli.git"
+build: [
+  ["obuild" "configure"]
+  ["obuild" "build" "lib-minicli"]
+]
+install: ["obuild" "install"]
+remove: ["ocamlfind" "remove" "minicli"]
+depends: [
+  "ocamlfind"
```

Comment:
this is likely a `{build}`-only dependency, or am I wrong here?

Reply:
yes, corrected